src/redux/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+// categoriesSlice.js
+
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: 'Under construction',
+  reducers: {
+    getCategoryStatus: (state) => state,
+  },
```

Comment:
- [ ] Kindly make sure your `inistialState` should start always empty and make sure it will change only in the reducer. Please make sure to start with the empty array and change the state only in the reducer.